extensions/amp-analytics/OWNERS.yaml

```diff
@@ -1,7 +1,4 @@
 # For an explanation of the OWNERS.yaml rules and syntax, see:
 # https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
 
-- avimehta
-- dvoytenko
-- lannka
-- zhouyx
+- wg-analytics
```

Comment:
let's keep Avi. We can remove Dima from the file here because they're the root owner.

Reply:
LGTM